.github/workflows/ci.yml

```diff
@@ -44,8 +48,11 @@ jobs:
     strategy:
       matrix:
         gradle-versions:
-          - 5.3,5.6
-          - 6.0,6.6
+          - 5.6
+          - 6.0,6.1,6.2
+          - 6.3,6.4,6.5
+          - 6.6,6.7,6.8
+          - '7.0'
```

Comment:
why is this one in single quotes, and the other ones not?

Reply:
Because it was being interpreted as a number by yaml.
Since `7.0` is `7`, that was being passed along to Gradle, which messed up the selection of which versions the integration tests are run on.